vllm/config.py

```diff
@@ -122,6 +122,8 @@ class ModelConfig:
             can not be gathered from the vllm arguments. 
         config_format: The config format which shall be loaded.
             Defaults to 'auto' which defaults to 'hf'.
+        processor_kwargs: Arguments to be forwarded to the model's processor,
```

Comment:
From my understanding, this only applies to multimodal processors at the moment, not the tokenizer. If that's the case, let's rename this to `mm_processor_kwargs` to make that more explicit

Reply:
Sounds good! I've renamed `processor_kwargs` -> `mm_processor_kwargs` and updated the engine CLI arg to make it more clear that it's for multimodal!